### PR TITLE
generate invite link

### PR DESCRIPTION
**Pull request by jermsam**

### CODE REVIEW

File: src/Editor.tsx [hunk 1]
```diff
@@ -35,11 +35,11 @@ export const Editor = ({doc, changeDoc}) => {
 
                                 changeDoc(
                                     (doc) => {
-                                        const layerToMove = doc.layers.splice(key, 1)[0]; 
+                                        const layerToMove = doc.layers.splice(key, 1)[0];
                                         doc.layers.splice(key-1, 0, layerToMove);
+                                        console.log(Array.from(doc.layers))
```
Comment (neftaly) [inline src/Editor.tsx]: Can you please get rid of this (and the other console.log below)?

**Review by neftaly — COMMENTED** [review]
lookd good aside from console.log

File: src/Editor.tsx (hunk 1 above)
Comment (jermsam) [reply src/Editor.tsx]: removed

File: package.json [hunk 2]
```diff
@@ -19,6 +19,9 @@
     "automerge-repo-react-hooks": "^0.0.54",
     "automerge-repo-storage-localforage": "^0.0.56",
     "eventemitter3": "^5.0.1",
+    "i": "^0.3.7",
+    "npm": "^9.7.2",
```
Comment (neftaly) [inline package.json:23]: I don't think "i" and "npm" are supposed to be here but we can can clean them up later with "depcheck", cause there might be other dependencies we're not using.

Comment (jermsam) [reply package.json:23]: yeah ... let me remove those ones

**Pull request merged**